pages/common/nm.md

```diff
@@ -0,0 +1,19 @@
+# nm
+
+> List symbol names in object files
+
+- List global (extern) functions in a file (prefixed with T)
+
+`nm -g {{file.o}}`
+
+- Demangle C++ symbols (make them readable)
+
+`nm {{file.o}} | c++filt`
```

Comment:
Or you could add a small note in the description with the workaround, although that would probably be frowned upon by some maintainers (cf. #321)